target/linux/x86/config-6.1

```diff
@@ -60,6 +60,7 @@ CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_IDLE=y
 CONFIG_CPU_IDLE_GOV_LADDER=y
+CONFIG_CPU_MITIGATIONS=y
```

Comment:
You can remove `SPECULATION_MITIGATIONS` from the configuration now.

Reply:
Yes，`CONFIG_SPECULATION_MITIGATIONS` It should be deleted.